diff --git a/google/cloud/bigtable/BUILD.bazel b/google/cloud/bigtable/BUILD.bazel
index 25549c8b02..f6f71e6b06 100644
--- a/google/cloud/bigtable/BUILD.bazel
+++ b/google/cloud/bigtable/BUILD.bazel
@@ -47,8 +47,7 @@ cc_library(
         "//conditions:default": [],
     }),
     visibility = [
-        ":__subpackages__",
-        "//:__pkg__",
+        "//visibility:public",
     ],
     # Do not sort: grpc++ must come last
     deps = [
