diff --git a/MODULE.bazel b/MODULE.bazel
index aa4f267671..051db01d86 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -23,7 +23,7 @@ module(
 bazel_dep(name = "platforms", version = "0.0.11")
 bazel_dep(name = "bazel_skylib", version = "1.7.1")
 bazel_dep(name = "rules_cc", version = "0.0.17")
-bazel_dep(name = "abseil-cpp", version = "20240722.1", repo_name = "com_google_absl")
+bazel_dep(name = "abseil-cpp", version = "20250512.1", repo_name = "com_google_absl")
 bazel_dep(name = "protobuf", version = "29.4", repo_name = "com_google_protobuf")
 bazel_dep(name = "boringssl", version = "0.0.0-20230215-5c22014")
 bazel_dep(name = "grpc", version = "1.69.0", repo_name = "com_github_grpc_grpc")
diff --git a/google/cloud/bigtable/BUILD.bazel b/google/cloud/bigtable/BUILD.bazel
index 25549c8b02..f6f71e6b06 100644
--- a/google/cloud/bigtable/BUILD.bazel
+++ b/google/cloud/bigtable/BUILD.bazel
@@ -47,8 +47,7 @@ cc_library(
         "//conditions:default": [],
     }),
     visibility = [
-        ":__subpackages__",
-        "//:__pkg__",
+        "//visibility:public",
     ],
     # Do not sort: grpc++ must come last
     deps = [
